### ISSUE TITLE
Support range requests

### ISSUE THREAD

**Issue by MarcelloTheArcane** [gmusicproxy/gmusicproxy#133]
At the moment, if I try to seek to a position with an HTML audio element, it restarts the audio.

Could HTTP range requests be supported, so that I can seek to places in music?

**Comment by MarcelloTheArcane** [comment]
Some handy docs:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests

1. Add the `Accept-Ranges: bytes` header to media requests
2. Return `Content-Range: 0-/<total>` header by default, with a `200` status
3. If a `Range` header is present on the request, return the appropriate ranges with a `206` status and `Content-Range: <start>-<end>/<total>` header.

It should be possible to set up round about [here in the code](https://github.com/gmusicproxy/gmusicproxy/blob/master/GMusicProxy#L292), I'm not too sure how to implement it.

https://github.com/gmusicproxy/gmusicproxy/blob/47eb52ef825061db797dfead11e5a1755daeaa9a/GMusicProxy#L285-L293

**Comment by jdwoody** [comment]
We could make this work, I've noticed this issue from time to time when navigating in volumio and/or mpd. The get_song method can use some clean up and I'm focusing on the YT music migration, but will work this in as time allows.

**Comment by MarcelloTheArcane** [comment]
@jdwoody Are updates pretty much on hold till we have the YT music integration?

**Comment by jdwoody** [comment]
No, updates are not on hold. I'll be cutting releases. I was looking at moving to Flask for serving content, which provides easier route configuration,  Swagger docs, etc. I'm not sure how involved this will be. If you can provide some documentation/code that would start play at a certain offset, I will try to work it in.

**Comment by MarcelloTheArcane** [comment]
I'm not that confident coding with Python, so this is  something I've found - not sure if you can make something of it: https://gist.github.com/lizhiwei/7885108
